rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/BatchOperationResult.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Bosch Software Innovations GmbH, 2018.
+ * Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.sw360.rest.resourceserver.core;
+
+import org.springframework.http.HttpStatus;
+
+public class BatchOperationResult {
```

Comment:
Maybe rename to a more 'generic' name like `MultiStatus` ?

Reply:
Changed the class name to `MultiStatus`